qt-models/maplocationmodel.cpp

```diff
@@ -9,8 +9,6 @@ const char *MapLocation::PROPERTY_NAME_NAME       = "name";
 MapLocation::MapLocation()
 {
 	m_uuid = 0;
-	m_name = "";
-	m_coordinate = QGeoCoordinate();
```

Comment:
Since the CID mentioned is only about m_uuid, I'd prefer if we squash this commit with the one that introduced these two initializations.
Happy to do this manually when merging.

Reply:
yes, as thinking about squash and force push ... will do.

Reply:
It's easy to fix if I merge locally instead of on GitHub, but if you are happy to do it in your tree and then force push, that of course is even easier for me :-)

Reply:
already done.